pkg/utils/gnmiutils.go

```diff
@@ -371,6 +375,12 @@ func UpdateForElement(value interface{}, path string, pathParams ...string) (*gn
 								IntVal: reflectValue.Index(i).Interface().(int64),
 							},
 						}
+					case reflect.Float32, reflect.Float64:
+						val = &gnmi.TypedValue{
+							Value: &gnmi.TypedValue_DoubleVal{
```

Comment:
I think you could have split this up in to `FloatVal` for `float32` and `DecimalVal` for ``float64`

Reply:
@SeanCondon both FloatVal and DecimalVal are deprecated, and replaced by DoubleVal:
https://github.com/openconfig/gnmi/blob/master/proto/gnmi/gnmi.pb.go#:~:text=type%20TypedValue_FloatVal%20struct%20%7B

Reply:
Hi @frouzbeh - OK I was not aware of this - it is unfortunate, since float and decimal are still 2 distinct types in the YANG language.

I think the quickest solution is to ignore the deprecation and use the solution I've suggested above, and then in parallel expand the onos-api proto defintion of Value Types